children/tests/snapshots/snap_test_api.py

```diff
@@ -154,30 +154,42 @@
     "data": {
         "children": {
             "edges": [
-                {"node": {"lastName": "0"}},
                 {"node": {"lastName": "1"}},
                 {"node": {"lastName": "2"}},
                 {"node": {"lastName": "3"}},
                 {"node": {"lastName": "4"}},
+                {"node": {"lastName": "5"}},
             ]
         }
     }
 }
 
 snapshots["test_children_offset_pagination[2-2] 1"] = {
-    "data": {"children": {"edges": [{"node": {"lastName": "4"}}]}}
```

Comment:
This was certainly not right, because the limit was set to 2 and only 1 item was returned!